Aspire Budgeting/ViewModels/FileSelectorViewModel.swift

```diff
@@ -0,0 +1,19 @@
+//
+// FileSelectorViewModel.swift
+// Aspire Budgeting
+//
+
+import Foundation
+
+protocol FileSelectorViewModelDelegate {
+  func setSelected(file: File)
+}
+
+struct FileSelectorViewModel {
+  let files: [File]
+  let delegate: FileSelectorViewModelDelegate
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/aspirebudgetingmobile/aspirebudgeting_ios/blob/fd43c30d9ee7d075625997a607bdb1dd002cf1a8/Aspire Budgeting/ViewModels/FileSelectorViewModel.swift#L14"></span>Delegates should be weak to avoid reference cycles.<br />
<code>weak_delegate</code> <code>FileSelectorViewModel.swift:14</code></td>
    </tr>
  </tbody>
</table>